custom_metrics/almanac.js

```diff
@@ -694,4 +694,190 @@ return JSON.stringify({
   'length_of_h1s': (() => {
     return [...document.querySelectorAll('h1')].map(node => node.innerText.length);
   })(),
+
+  'a11y': (() => {
```

Comment:
Can you create a new `accessibility.js` file for these metrics? At a glance it doesn't look like they depend on any of the "global" functions within `almanac.js`.